### ISSUE TITLE
NPM pointing to v0.22 which does not have installer packages

### ISSUE THREAD

**Issue by alabama** [avencera/rustywind#112]
Hey. 

When installing rustywind without a specific version it tries to install latest which is v0.22. This version does only have one installer package for `aarch64` linux systems. The executable just fails with: `rustywind: line 1: empty: command not found`

The npm packge refers to v0.22 as @latest 
![image](https://github.com/user-attachments/assets/7b7775d3-62a0-4e22-aad0-ba96620ffde7)

Github right now
![image](https://github.com/user-attachments/assets/b54aee20-cbcc-4c26-b3b7-7c69bb48bb86)

Could you bump the version in NPM to v0.23.1 which seems to have all the installers ready?

**Comment by elalemanyo** [comment]
@alabama I believe it is related to an [issue](https://github.com/avencera/rustywind/issues/106) from the past.

**Comment by alabama** [comment]
good point @elalemanyo but if I specify the exact version I want to install from github everything works fine:

![image](https://github.com/user-attachments/assets/955e21f0-12a8-4cfa-9b1b-46a0a1addf67)
+
![image](https://github.com/user-attachments/assets/4f687882-a660-4d55-9044-fbd8999a041d)

**Comment by praveenperera** [comment]
sorry about that its fixed,  installing from npm works again
